app/javascript/react/containers/FormContainer.js

```diff
@@ -31,15 +31,21 @@ class FormContainer extends Component {
   handleDescriptionChange(event){
       this.setState({ description: event.target.value })
   }
+
   handleSubmit(event){
     event.preventDefault();
-    let formPayload = {
-      rating: this.state.rating,
-      description: this.state.description,
-      vote_count: 0
+    if(this.state.user !== null){
+      let formPayload = {
+        rating: this.state.rating,
+        description: this.state.description,
+        vote_count: 0
+      }
```

Comment:
You should not be setting vote_count here. If you need to handle having a vote count for a review, that should be set on the backend